arm-web/compositeWebAppClient.json

```diff
@@ -4,15 +4,15 @@
     "description": "Composite Swagger for WebSite Management Client"
   },
   "documents": [
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2015-08-01/AppServiceCertificateOrders.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2016-09-01/AppServiceEnvironments.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2016-09-01/AppServicePlans.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2016-03-01/Certificates.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2015-04-01/Domains.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2016-03-01/Recommendations.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2016-03-01/ResourceProvider.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2015-04-01/TopLevelDomains.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2016-08-01/WebApps.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-web/2016-03-01/DeletedWebApps.json"
+    "./arm-web/2015-08-01/AppServiceCertificateOrders.json",
```

Comment:
"./arm-web" is incorrect. It should start with the api-version folder, since the composite doc is already present in the arm-web folder.